apps/api/src/applicant/ien-master.service.ts

```diff
@@ -31,12 +31,7 @@ export class IENMasterService {
   ) {}
 
   async getStatus(): Promise<IENApplicantStatus[]> {
-    return this.ienApplicantStatusRepository.find({
-      where: {
-        parent: IsNull(),
-      },
-      relations: ['children'],
-    });
+    return this.ienApplicantStatusRepository.find();
```

Comment:
this changes the returned data structure and might cause troubles on FE components.

Reply:
It certainly did, I had to make some changes to the FE to accommodate this. Right now it is stable, but I expect we will find a number of bugs related to this. 
I am hoping that once this is merge, we can tackle them as a team.